.github/CODEOWNERS

```diff
@@ -522,29 +525,33 @@ x-pack/examples/files_example @elastic/kibana-app-services
 /x-pack/plugins/security_solution/server/lib/detection_engine/routes/index @elastic/security-detections-response-alerts
 /x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals @elastic/security-detections-response-alerts
 
-
 ## Security Solution sub teams - Detections and Response Rules
-/x-pack/plugins/security_solution/cypress/integration/detection_rules @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/common/detection_engine/rule_monitoring @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/common/detection_engine/schemas/common @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/common/detection_engine/schemas/request @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/common/detection_engine/schemas/response @elastic/security-detections-response-rules
 
-/x-pack/plugins/security_solution/public/detections/components/rules @elastic/security-detections-response-rules
-/x-pack/plugins/security_solution/public/detections/components/severity @elastic/security-detections-response-rules
-/x-pack/plugins/security_solution/public/detections/components/status @elastic/security-detections-response-rules
-/x-pack/plugins/security_solution/public/rules @elastic/security-detections-response-rules
 /x-pack/plugins/security_solution/public/common/components/health_truncate_text @elastic/security-detections-response-rules
 /x-pack/plugins/security_solution/public/common/components/links_to_docs @elastic/security-detections-response-rules
-/x-pack/plugins/security_solution/public/common/components/callouts @elastic/security-detections-response-rules
 /x-pack/plugins/security_solution/public/common/components/ml_popover @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/public/common/components/popover_items @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/public/detection_engine/rule_monitoring @elastic/security-detections-response-rules
 /x-pack/plugins/security_solution/public/detections/components/callouts @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/public/detections/components/modals/ml_job_upgrade_modal @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/public/detections/components/rules @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/public/detections/components/rules/rule_preview @elastic/security-detections-response-alerts
```

Comment:
@marshallmain @e40pud So rule preview goes to the Alerts area!

Is it possible to move `x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/use_preview_rule.ts` to `x-pack/plugins/security_solution/public/detections/components/rules/rule_preview`?

Reply:
Yup, we can do that!

Reply:
@banderror here is [the PR](https://github.com/elastic/kibana/pull/140758)